### ISSUE TITLE
Ensure SpelExpressionConverterConfiguration is equally configured during single and/or multi-binder configurations 

### ISSUE THREAD

**Issue by olegz** [spring-cloud/spring-cloud-stream#1412]
See the original issue originated in Kafka binder [here](https://github.com/spring-cloud/spring-cloud-stream-binder-kafka/issues/408)

Currently for multi-binder cases we do NOT set parent context to a binder context, thus making these type of configurations invisible to the binder's context. While the issue has been identified specifically with `SpelExpressionConverterConfiguration`, there may be other similar once.

**Comment by VasuBhaskar** [comment]
@olegz any idea on when this fix will be available. I am waiting for this to complete a deliverable :-)

**Comment by olegz** [comment]
Sure, we'll be releasing 2.0.1 shortly (few days) and it will include the fix (PR is already out there).

**Comment by VasuBhaskar** [comment]
Thanks a lot @olegz . hope "few days" is sooner for me 👍

**Comment by olegz** [comment]
@VasuBhaskar you can actually help us to validate by simply building a snapshot from this [PR](https://github.com/spring-cloud/spring-cloud-stream/pull/1413) or this [branch](https://github.com/olegz/spring-cloud-stream/tree/GH-1412) (same thing) and applying to your app.

**Comment by VasuBhaskar** [comment]
@olegz working on it now. Will keep you posted.

**Comment by VasuBhaskar** [comment]
@olegz Good news is that the fix is working!! 

But I had to do the following additional things, so just thought of letting you know.

- Added these dependencies to my app even after including the snapshot, because I was getting classnotfound error 
```
compile group: 'org.springframework', name: 'spring-tuple', version: '1.0.0.RELEASE'
compile group: 'com.esotericsoftware', name: 'kryo', version: '4.0.2'
```

- import example.avro.User; Somehow I was not able to access the avro schema declared in the resources. I commentedout the entire AvroMessageConverterSerializationTests

**Comment by olegz** [comment]
You may have some gradle cache issues, but yes we were able to validate on our side that the fix is working. 
In any event glad to hear it's moving in the right direction.

**Comment by VasuBhaskar** [comment]
@olegz am sorry for this question on a closed issue, but what should be the version , I should reference to get this fix. I am confused on this. Please help me out

**Comment by sobychacko** [comment]
@VasuBhaskar Elmhurst.SR1 is getting released today that contains 2.0.1 versions of the core and the binders. This fix will be in them.

**Comment by sobychacko** [comment]
@VasuBhaskar Elmhurst.SR1 release is now available through maven central. Release notes are here: https://github.com/spring-cloud/spring-cloud-stream-starters/releases/tag/vElmhurst.SR1
